### PR TITLE
added navigation

### PR DESCRIPTION
**Pull request by sanjay-xdr**
added navigation 
demo :- https://loving-mestorf-0a3389.netlify.app/component/navigation/navigation

### CODE REVIEW

File: Component/Navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="../../Utility CSS/utility.css" />
+    <link rel="stylesheet" href="./navigation.css" />
+
```
Comment (HrishikeshMahalle) [inline Component/Navigation/navigation.html:15]: try to add Different css in single css and import it.

**Review by HrishikeshMahalle — APPROVED** [review]
Great job, Good to merge

File: Component/Navigation/navigation.html (hunk 1 above)
Comment (sanjay-xdr) [reply Component/Navigation/navigation.html:15]: yes in the end i will raise a PR regarding code refactor

**Review by harshparab — APPROVED** [review]
Looks good to merge

**Pull request merged**